.gitpod.yml

```diff
@@ -0,0 +1,2 @@
+tasks:
+  - init: mvn install -DskipTests=false
```

Comment:
Why are you enabling the test cases? I would have expected the opposite since these take quite a while when starting up the IDE.

Reply:
This was auto-generated. I agree it makes sense to set this to true.